backend-shared/config/db/deploymenttoapplicationmapping.go

```diff
@@ -329,3 +329,20 @@ func (obj *DeploymentToApplicationMapping) GetAsLogKeyValues() []interface{} {
 		"dtamNamespace", obj.DeploymentNamespace, "dtamUID", obj.Deploymenttoapplicationmapping_uid_id,
 		"dtamNamespaceUID", obj.NamespaceUID}
 }
+
+// Get deploymentToApplicationMappings in a batch. Batch size defined by 'limit' and starting point of batch is defined by 'offSet'.
+// For example if you want deploymentToApplicationMappings starting from 51-150 then set the limit to 100 and offset to 50.
+func (dbq *PostgreSQLDatabaseQueries) GetDeploymentToApplicationMappingBatch(ctx context.Context, deploymentToApplicationMappings *[]DeploymentToApplicationMapping, limit, offset int) error {
+	err := dbq.dbConnection.
+		Model(deploymentToApplicationMappings).
+		Order("seq_id ASC").
+		Limit(limit).   // Batch size
+		Offset(offset). // offset+1 is starting point of batch
+		Context(ctx).
+		Select()
+
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
```suggestion
	return err
```

Nit, feel free to ignore